template.yaml

```diff
@@ -228,8 +230,11 @@ Resources:
       Handler: sendEmailRequestToNotify.handler
       Environment:
         Variables:
+          ENVIRONMENT: !Ref Environment
           NOTIFY_API_SECRETS_ARN: '{{resolve:ssm:NotifySecretSetArn}}'
           CLOSE_TICKET_QUEUE_URL: '{{resolve:ssm:CloseZendeskTicketQueueUrl}}'
+          USE_NOTIFY_MOCK_SERVER: !If [IsStubbedEnvironment, 'true', 'false']
+          MOCK_SERVER_BASE_URL: 'https://mockserver.transaction.build.account.gov.uk'
```

Comment:
Does it make sense for this to be a SSM::Parameter

Reply:
It could be, but it would have to be manually set, because the SSM parameter for this url is in the audit account, unfortunately.